Destiny.ScrimTracker.App/Controllers/MatchController.cs

```diff
@@ -32,55 +32,39 @@ public IActionResult Index()
         }
 
         [Authorize]
-        [Route("match")]
-        public IActionResult AddMatchView()
-        {
-            return View();
-        }
-        
-        [Authorize]
-        [Route("add_match")]
-        public IActionResult CreateMatchForm([FromQuery] int numOfTeams, [FromQuery] int playersPerTeam)
+        [Route("new")]
+        public IActionResult CreateMatchForm()
         {
             var guardians = _guardianService.GetGuardians();
             ViewData.Add("Guardians", guardians.OrderBy(g => g.Guardian.GamerTag).Select(g => g.Guardian.GamerTag));
-            ViewData.Add("NumberOfTeams", numOfTeams);
-            ViewData.Add("PlayersPerTeam", playersPerTeam);
 
             var match = new CreateMatchFormModel();
-
-            var teams = new List<Team>();
-            for (var i = 0; i < numOfTeams; i++)
-            {
-                teams.Add(new Team());
-                teams[i].MatchResults = new List<MatchResult>();
-                for (var j = 0; j < playersPerTeam; j++)
-                {
-                    teams[i].MatchResults.ToList().Add(new MatchResult());
-                }
-            }
-
-            match.Teams = teams;
             return View(match);
         }
 
         [HttpPost]
         [Authorize]
-        [Route("add_match")]
+        [Route("new")]
         public IActionResult FormatRequestAndAddMatch([FromQuery] int numOfTeams, [FromQuery] int playersPerTeam)
         {
             var createMatchRequest = new CreateMatchRequest();
 
             createMatchRequest.MatchType = Enum.Parse<MatchType>(Request.Form["MatchType"]);
+
+            var formKeys = Request.Form.Keys;
+
+            Console.WriteLine($"Key count: {formKeys.Count()}");
             var teams = new List<MatchTeam>();
-            for (var i = 0; i < numOfTeams; i++)
+            for (var i = 0; i < 2; i++)
```

Comment:
Couldn't figure out how to make the form reactive enough to be able to add team forms on the fly. Added an issue to track it. For now, since two teams is our primary use case, i figured this would do?